sonar-ws/src/main/protobuf/ws-issues.proto

```diff
@@ -42,7 +42,6 @@ message SearchWsResponse {
   repeated Component components = 7;
   optional sonarqube.ws.commons.Rules rules = 8;
   optional sonarqube.ws.commons.Users users = 9;
-  optional ActionPlans actionPlans = 10;
```

Comment:
deprecated fields should stay in order to know why we can't used these numbers => we should renamed these fields by prefixing them with unusedXXX (see what Simon has done for characteristic in ws-rules.proto)